main.py

```diff
@@ -81,6 +81,27 @@ def handle_dnf(dictionary):
         output += f'{dnfer_row.name} now has {dnfer_row.dnf_count} dnfs\n'
     return output
 
+LAME_THRESHOLD = 5
+def handle_lames(dictionary):
+    # pull up all users
+    # lames are users that are in set of all users
+    # but not in (input) dictionary
+    participants = dictionary.values()
+    lames = []
+    for user in session.query(User).all():
+        # if they particpated, reset their lame count
+        # otherwise, add 1 lame
+        if user.name in participants:
+            user.lame_count = 0
+        else:
+            user.lame_count += 1
+            # check if they've reached the lame threshold
+            # add them to the output if so
+            if user.lame_count > LAME_THRESHOLD:
+                lames.append(user.name)
+    output = f"Prepare to be 🦵: {', '.join(lames)} \n"
```

Comment:
nit: single quote

Reply:
non blocking

Reply:
Needs the double quotes (yeah sucks) since we're using single quotes within the string.  Weird that having the brackets around them doesn't "escape" them though. @Guido Van Rossum feature request in `3.10` 🙋‍♂

Reply:
Resolving